sdk/src/test/java/io/dapr/actors/client/ActorProxyImplTest.java

```diff
@@ -4,18 +4,290 @@
 import io.dapr.actors.runtime.ActorStateSerializer;
 import org.junit.Assert;
 import org.junit.Test;
+import org.mockito.Mockito;
+import reactor.core.publisher.Mono;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 public class ActorProxyImplTest {
 
     @Test()
     public void constructorActorProxyTest() {
-        final ActorProxyHttpAsyncClient actorProxyAsyncClient = (ActorProxyHttpAsyncClient)new ActorProxyClientBuilder().buildAsyncClient();
-        final ActorProxyImpl actorProxy= new ActorProxyImpl(
-          "myActorType",
-          new ActorId("100"),
-          new ActorStateSerializer(),
-          actorProxyAsyncClient);
-        Assert.assertEquals(actorProxy.getActorId().toString(),"100");
-        Assert.assertEquals(actorProxy.getActorType(),"myActorType");
+        final ActorProxyHttpAsyncClient actorProxyAsyncClient = mock(ActorProxyHttpAsyncClient.class);
+        final ActorStateSerializer serializer = mock(ActorStateSerializer.class);
```

Comment:
Please, use a real serializer. It is OK since it does not do IO and it must uncover more problems with this implementation.

Reply:
Agree, but this is just to test the constructor, that is that I  inject the mock, which is just for this test.

Reply:
OK.